openamp/overview.rst

```diff
@@ -28,6 +30,9 @@ There are some AMP fundamentals which influence the OpenAMP Architecture choices
 * **IPC**: `Inter Processor Communications <https://en.wikipedia.org/wiki/Inter-process_communication>`_ needs to be established to enable communication and control.
 * **Resource Isolation**: AMP systems can be supervised (using a hypervisor) or unsupervised.
```

Comment:
AMP systems can be supervised (**e.g** using a hypervisor) or unsupervised.

Reply:
changed